docs/_includes/breadcrumb.html

```diff
@@ -0,0 +1,49 @@
+<div id="breadcrumb">
+
+    <h2><span class="secondary-text">@ebay/skin/</span>breadcrumb</h2>
+
+    <h3>Breadcrumb - Last item as the parent</h3>
+    <p>The last item in the breadcrumb list is the parent of the current page. Use this pattern if the current page category is not specific</p>
+
+    <div class="demo">
+        <div class="demo__inner">
+            <ul class="breadcrumb">
+                <li><a href="https://www.ebay.com/">ebay</a></li>
+                <li><a href="https://www.ebay.com/rpp/cell-phone-pda">Cell Phones, Smart Watches & Accessories</a></li>
+                <li><a href="https://www.ebay.com/b/Smart-Watch-Accessories/182064/bn_16565905">Smart Watch Accessories</a></li>
+                <li><a href="https://www.ebay.com/b/Smart-Watch-Bands/182068/bn_16565906">Smart Watch Bands</a></li>
+            </ul>
+        </div>
+    </div>
+    {% highlight html %}
+    <ul class="breadcrumb">
+        <li><a href="https://www.ebay.com/">ebay</a></li>
+        <li><a href="https://www.ebay.com/rpp/cell-phone-pda">Cell Phones, Smart Watches & Accessories</a></li>
+        <li><a href="https://www.ebay.com/b/Smart-Watch-Accessories/182064/bn_16565905">Smart Watch Accessories</a></li>
+        <li><a href="https://www.ebay.com/b/Smart-Watch-Bands/182068/bn_16565906">Smart Watch Bands</a></li>
+    </ul>
+    {% endhighlight %}
+
+    <h3>Breadcrumb - Last item as the current</h3>
+    <p>Breadcrumbs indicate the steps or path that a user has taken to arrive at the current page. There is no need to make the last item of a breadcrumb list a link, as it is the current page being viewed.</p>
+
+    <div class="demo">
+        <div class="demo__inner">
+            <ul class="breadcrumb">
+                <li><a href="https://www.ebay.com/">ebay</a></li>
+                <li><a href="https://www.ebay.com/rpp/cell-phone-pda">Cell Phones, Smart Watches & Accessories</a></li>
+                <li><a href="https://www.ebay.com/b/Smart-Watch-Accessories/182064/bn_16565905">Smart Watch Accessories</a></li>
+                <li><a href="javascript:;" class='current'>Smart Watch Bands</a></li>
```

Comment:
All href values, if present, must be a valid url (i.e. they cannot be `javascript:`). In this case I would simply remove the href attribute. Also, instead of using `class=current`, please use `aria-current="page"`